en/components/radial-gauge.md

```diff
@@ -47,58 +57,13 @@ The Radial Gauge comes with styling flexibility through the various overrides co
 
```

Comment:
In styling add customization of the current value label and tick labels via the available text styles. And add a sentence that for the range brushes it is recommended to choose one of the data viz palette colors.